instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -454,10 +509,14 @@ async def __call__(self, scope, receive, send):
                     span_name,
                     scope,
                     send,
+                    duration_attrs,
                 )
 
                 await self.app(scope, otel_receive, otel_send)
```

Comment:
I'm wondering if we want to start the timer right before the send? Otherwise we will be counting the time it takes to do the OpenTelemetry stuff like set attributes, request/response hook, etc.?

Reply:
added changes.